tracing-log/src/lib.rs

```diff
@@ -246,3 +256,136 @@ impl AsTrace for log::Level {
         }
     }
 }
+
+/// Extends log `Event`s to provide complete `Metadata`.
+///
+/// In `tracing-log`, an `Event` produced by a log (through `AsTrace`) has an hard coded
```

Comment:
Nit:
```suggestion
/// In `tracing-log`, an `Event` produced by a log (through [`AsTrace`]) has an hard coded
```
(and add a link reference to the bottom of the doc comment)